Modules/src/UserInputValidator.cc

```diff
@@ -44,6 +61,86 @@ namespace ChimeraTK {
     return rejected;
   }
 
+  void UserInputValidator::enableDeepValidation() {
+    // Find out accessors with return
+    for(auto& accessor : _module->getAccessorListRecursive()) {
+      if(accessor.getDirection() == VariableDirection{VariableDirection::feeding, true} &&
+          accessor.getModel().getTags().count(std::string(UserInputValidator::tagValidatedVariable)) > 0) {
+        _downstreamInvalidatingReturnChannels.emplace(accessor.getAppAccessorNoType().getId());
+      }
+    }
+
+    auto keepPvAccesWithReturnChannel = Model::EdgeFilter([](const Model::EdgeProperties& edge) -> bool {
```

Comment:
This one is a candidate for an official filter, I guess :-)

Reply:
Moved to Model.h and added to the list of filters